twweet_cli.py

```diff
@@ -56,7 +60,16 @@ def get_all_tweets(screen_name):
     outtweets = [[tweet.id_str,tweet.created_at,tweet.text.encode("utf-8")]for tweet in alltweets]
 
     # write to csv
-    fn=str(screen_name+'_tweets.csv')
+    global TweetsStorage
+    TweetsStorage=str(os.getcwd()+TweetsStorage)
+    os.makedirs(os.path.dirname(TweetsStorage), exist_ok=True)
+#    if not os.path.exists(os.path.dirname(TweetsStorage)):
+#        try:
+#            os.makedirs(os.path.dirname(TweetsStorage))
```

Comment:
These codes work for Python 2.7
Are there multiple branches for this project having Python 2.7. if yes then we can port these commented codes to that branche
The commented codes will work for Python 2.7